x-pack/platform/plugins/shared/fleet/public/applications/integrations/sections/epm/screens/detail/index.tsx

```diff
@@ -633,6 +633,8 @@ export function Detail() {
         href: getHref('integration_details_overview', {
```

Comment:
We could extract the params to an object to reuse, something like:
```
const pathValues = {
          pkgkey: packageInfoKey,
          ...(integration ? { integration } : {}),
          ...(returnAppId ? { returnAppId } : {}),
          ...(returnPath ? { returnPath } : {}),
        }),
      };
```

Reply:
Done!